policy-descriptor/examples/policy-example.yml

```diff
@@ -0,0 +1,23 @@
+# do NOT delete: this is used by tng-sdk-descriptorgen
+# default YAML description of an ubuntu vm
+
+
+---
+descriptor_schema: "https://raw.githubusercontent.com/sonata-nfv/tng-schema/master/policy-descriptor/policy-schema.yml"
+
+name: "samplePolicy"
+
+
+rules:
+  - name: "FirstRule"
+    duration: "to be defined"
+    condition: "to be defined"
+    action: "to be defined"
```

Comment:
In the schema, you define action to be an array of objects with mandatory `type` field. I guess, you should set up tests (as Manuel suggested) in order to ensure the descriptors match the schema (and vice versa).